scripts/n2n-httpd

```diff
@@ -456,13 +456,15 @@ def main():
     snrpc.debug = args.debug
     snrpc.defaultkey = args.key
 
-    signal.signal(signal.SIGPIPE, signal.SIG_DFL)
+    if hasattr(signal, 'SIGPIPE'):
+        signal.signal(signal.SIGPIPE, signal.SIG_DFL)
 
     socketserver.TCPServer.allow_reuse_address = True
     handler = functools.partial(SimpleHandler, rpc, snrpc)
 
     httpd = socketserver.TCPServer(("", args.port), handler)
     try:
+        print(f"Serving HTTP at port {args.port} (http://localhost:{args.port}/) ...")
```

Comment:
Do you really think this is necessary? The existing code is following the standard unix pattern of silence-as-success.

Reply:
Not necessary, but I do think it's better to let the user know what happened

Feel free to delete this if you don't like it